tests/select.spec.ts

```diff
@@ -222,12 +222,15 @@ test('On pressing enter over an option, that option should be selected', async (
   expect(await input.inputValue()).toEqual('');
   // press down over an option
   await page.keyboard.press('ArrowDown');
+  // const valueChanged = waitForCustomEvent(page, 'change');
```

Comment:
Should these lines be commented out? Also, do we want t listen for an `input` event here rather than `change`? (see other comment)

Reply:
I think we would want to listen to both since the value will be changed after it got selected on enter. Will fix the commented out part though